csharp-refactoring/RefactoringCodeBloaters/Tests/CalculationServiceTests.cs

```diff
@@ -0,0 +1,45 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using RefactoringCodeBloaters.LongMethod.Correct;
+
+namespace Tests
+{
+    [TestClass]
+    public class CalculationServiceTests
+    {
+        private readonly CalculationService calculationService;
+        public CalculationServiceTests()
+        {
+            calculationService = new CalculationService();
+        }
+
+        [TestMethod]
+        public void CalculateDiscount_LessThan250_ReturnsOriginalPrice()
```

Comment:
As stated in the guidelines, - 
test should follow the BDD convention: GivenX_WhenY_ThenZ
- the given part is optional, but the other two are not
- also, the delimiter (underline) should only be used between these sections, and not anywhere else
GivenPriceLessThan250_WhenCalculatingPriceAfterDiscount_ThenReturnOriginalPrice()

This applies to all Test methods as well.